packages/palette-docs/content/docs/tokens/Color.mdx

```diff
@@ -2,74 +2,74 @@
 name: Color
 lastPointOfContact: Owen
 ---
-## Primary colors
-
 
+## Primary colors
 
 <ColorComponent color="black100" />
 
-This is our primary brand color. It is used as the default text color on light backgrounds and as the primary color of our logo. 
+This is our primary brand color. It is used as the default text color on light
+backgrounds and as the primary color of our logo.
 
-- - -
+<Spacer m={6} />
```

Comment:
Maybe `<Spacer my={4} />`? Maybe a littleee less whitespace.

Reply:
:+1: 
![screencapture-localhost-8000-tokens-color-2019-04-22-16_45_13](https://user-images.githubusercontent.com/5361806/56529063-2672e900-651e-11e9-8919-8a5557ab5037.png)

Reply:
There we go 👍